packages/builders/scripts/fast-usdc/add-operators.build.js

```diff
@@ -0,0 +1,83 @@
+// @ts-check
+import { makeHelpers } from '@agoric/deploy-script-support';
+import { getManifestForAddOperators } from '@agoric/fast-usdc/src/add-operators.core.js';
+import { toExternalConfig } from '@agoric/fast-usdc/src/utils/config-marshal.js';
+import { configurations } from '@agoric/fast-usdc/src/utils/deploy-config.js';
+import { Far } from '@endo/far';
+import { parseArgs } from 'node:util';
+
+/**
+ * @import {CoreEvalBuilder, DeployScriptFunction} from '@agoric/deploy-script-support/src/externalTypes.js';
+ * @import {ParseArgsConfig} from 'node:util';
+ * @import {FastUSDCConfig, FeedPolicy} from '@agoric/fast-usdc/src/types.js';
+ * @import {FastUSDCOpts} from './init-fast-usdc.js';
+ */
+
+const { keys } = Object;
+
+/** @type {ParseArgsConfig['options']} */
+const options = {
+  net: { type: 'string' },
+  oracle: { type: 'string', multiple: true },
+};
+const oraclesUsage = 'use --oracle name:address ...';
+
+const crossVatContext = /** @type {const} */ ({
+  /** @type {Brand<'nat'>} */
+  USDC: Far('USDC Brand'),
+});
+
+/** @type {CoreEvalBuilder} */
```

Comment:
```suggestion
/** @satisfies {CoreEvalBuilder} */
```

Using `@type` hides type errors around `config`.

Maybe not critical, but probably worth fixing.